app/controllers/users.js

```diff
@@ -1,10 +1,13 @@
-const { createUser } = require('../services/users');
+const { User } = require('../models');
+const { generateUserToken } = require('../services/users.js');
 
-exports.createUser = async (req, res, next) => {
-  try {
-    const message = await createUser(req.query);
-    res.status(201).send(message);
-  } catch (err) {
-    next(err);
-  }
+module.exports = {
```

Comment:
I think `exports.createUser = () => {}` is better, or don't define the method inside`module.exports` because is difficult to read.

something like:

`const createUser = async (req, res, next) => {...};`
`const createUserToken = async (req, res, next) => {...};`

`module.exports = {`
`createUser,`
`createUserToken`
`}`